src/mscorlib/src/System/TypeLoadException.cs

```diff
@@ -103,7 +106,13 @@ public String TypeName
 
         protected TypeLoadException(SerializationInfo info, StreamingContext context) : base(info, context)
         {
-            throw new PlatformNotSupportedException();
+            if (info == null)
+                throw new ArgumentNullException(nameof(info));
```

Comment:
Is this needed?

Reply:
It was there before which means removing it is a breaking change :/ There are no checks anywhere else so this one is inconsistent.

Reply:
Doesn't the base ctor do this same check?

Reply:
Ah you are right. https://source.dot.net/#System.Private.CoreLib/src/System/Exception.cs,75 I'll remove it.